plugins/basic/python/tk_maya_plugin_basic/plugin_engine.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+
+# For now, import the Shotgun toolkit core included with the plug-in,
+# but also re-import it later to ensure usage of a swapped in version.
+import sgtk as tk_core
```

Comment:
I think this will be confusing for most toolkit devs. Can't we just call it sgtk?